_posts/2019-01-08-data-driven-static-resume.adoc

```diff
@@ -0,0 +1,202 @@
+----
+
+The first and last lines are conditionals; they ensure the enveloped settings are only applied during rendering procedures if the `backend-pdf` setting (“attribute” in AsciiDoc parlance) is defined, but they will replace any previously set parameters with the same names.
+
+Which gets us back to the LiquiDoc config.
+The next stage instructs LiquiDoc to actually render the PDF version of the output from the source file prebuilt in the last stage (`_build/pages/resume.adoc`).
+
+.From _configs/build-global.yml -- LiquiDoc config file resume PDF render stage
+[source,yaml]
+----
+include::assets/includes/build-global.yml[tags="render-resume-example-snippet"]
+----
+
+This block invokes a very simple Asciidoctor conversion using the link:https://github.com/asciidoctor/asciidoctor-pdf[PDF backend extension], which engages Prawn for the final conversion.
+A customized link:https://github.com/briandominick/codewriting/blob/master/theme/pdf/resume-theme.yml[YAML-formatted theme file] enforces the style of the PDF output.
+
+The resulting PDF artifact will later be copied into the Jekyll-generated website, but first we have to perform that Jekyll generation.
+The LiquiDoc config has us covered here, as well.
+
+[[example-config-jekyll]]
+.From _configs/build-global.yml -- Jekyll build execution stage
+[source,yaml]
+----
+include::assets/includes/build-global.yml[tags="build-jekyll-example-snippet"]
+----
+
+This section of the config script instructs Jekyll to generate a site, one page of which will be the HTML version of the resumé, as dictated by the `_build/pages/resume.adoc` file's frontmatter settings referenced above.
+This conversion does not require a specific instruction, as Jekyll renders all files in `_build/pages/` by default, and this prebuilt file is after all just another Jekyll-ready AsciiDoc file like link:https://github.com/briandominick/codewriting/tree/master/pages[all its siblings].
+
+== Conclusion
+
+These are all the main features of an AJYL platform: small data in YAML flat files, content in AsciiDoc, templating and transformation with Liquid, and Jekyll to put it all together as a proper website.
+
+The rest of the Codewriting site uses similar techniques to massage data and content together, either during prebuilding with Liquid or when rendering final artifacts.
+Asciidoctor uses parameters (AsciiDoc attributes) when converting AsciiDoc files to HTML, PDF, and other formats.
+Also during the render build, Jekyll engages Liquid's template engine to construct the site's HTML (or even to link:https://github.com/asciidoctor/jekyll-asciidoc#enabling-liquid-preprocessing[preprocess AsciiDoc content files containing Liquid markup]).
+All this complementarity is key to the AJYL environment.
+
+While a resumé is merely a novel implementation of this varied and powerful technology stack, imagine what this kind of flexibility can do for conveying complex product data in numerous permutations of editions and output formats.
```

Comment:
Move this to the top. Rad.

Reply:
I don't want to leave the blog without a good conclusion sentence, and I don't have it in me to try to come up with another haha. But thanks!